components/features.tsx

```diff
@@ -43,340 +95,294 @@ export default function Features() {
+                svg={
+                  <svg
+                    className="w-6 h-6 fill-primary-300"
+                    viewBox="0 0 12 12"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M11.854.146a.5.5 0 00-.525-.116l-11 4a.5.5 0 00-.015.934l4.8 1.921 1.921 4.8A.5.5 0 007.5 12h.008a.5.5 0 00.462-.329l4-11a.5.5 0 00-.116-.525z"
+                      fillRule="nonzero"
+                    />
+                  </svg>
+                }
+                heading={'Test Deduplication'}
+                description="Automatically detect and remove duplicate tests, ideal for scenarios recorded in live-environments."
+              />
+            </div>
+            <div className="flex items-center detail">
+              <TextSection
+                svg={
+                  <svg
+                    className="w-6 h-6 fill-primary-300"
+                    viewBox="0 0 12 12"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M11.334 8.06a.5.5 0 00-.421-.237 6.023 6.023 0 01-5.905-6c0-.41.042-.82.125-1.221a.5.5 0 00-.614-.586 6 6 0 106.832 8.529.5.5 0 00-.017-.485z"
+                      fillRule="nonzero"
+                    />
+                  </svg>
+                }
+                heading={'Native Integration'}
+                description="Merge Keploy tests with testing libraries(JUnit, go-test, py-test, jest) for combined test coverage."
+              />
+            </div>
           </div>
-          <div>
-            <a
-                className="btn text-white bg-secondary-300 hover:font-semibold w-full sm:w-auto sm:ml-4"
-                href="https://calendar.app.google/8Ncpff4QnAhpVnYd8"
-            >
-              Book Cloud Demo
-            </a>
+          <div className="flex items-center [&>*]:absolute relative  justify-center h-screen right-content">
+            <TestAndStubGenerationImage />
+            <TestDuplicationImage />
+            <div className="inline-flex flex-col imageToShow top-[30vh]">
+              <Image
+                className="w-full h-auto mx-auto rounded md:max-w-none"
+                src={NativeIntegration}
+                width={400}
+                height="462"
+                alt="Features bg"
+              />
+              {/*<Image className="absolute left-0 w-full transform md:max-w-none animate-float" src={FeaturesElement} width={500} height="44" alt="Element" style={{ top: '30%' }} />*/}
+              {/*<div className="absolute left-0 w-full text-left transform md:max-w-none animate-float" style={{ top: '100%' }}><CopyButton codeToCopy={"curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh"} /></div>*/}
+            </div>
           </div>
         </div>
-      </div>
-    </section>
+        <div className="max-w-6xl mx-auto mb-24 -mt-24 text-center">
```

Comment:
Hey @sahadat-sk everything looks fine just remove this division it is not going well with the UI

Reply:
Done